src/hooks/useTracking.js

```diff
@@ -0,0 +1,131 @@
+import { useRef, useEffect } from 'preact/hooks'
+
+const SCHEMA_NAME = 'InukaPageView'
+const SCHEMA_REV = 19739286
+const USER_ID_KEY = 'INUKA-PV-U'
+const SESSION_ID_KEY = 'INUKA-PV-S'
+const ONE_HOUR = 3600 * 1000
+const BASE_URL = 'https://en.wikipedia.org/beacon/event'
```

Comment:
Should this URL vary based on current language?

Reply:
Event Logging uses `webHost` `wiki` to determine the article language, upon checking with team there is no need to change the base url.